### ISSUE TITLE
Holes while camera is moving

### ISSUE THREAD

**Issue by lilleyse** [CesiumGS/cesium-omniverse#255]
Fabric prims don't finish loading until the camera stops moving, which results in holes.

https://user-images.githubusercontent.com/915398/230969896-43627ef8-a840-4905-a5ac-0638ad0bde61.mp4

<br>

I've tried a few different options to force synchronous loading, but no luck so far:

```
app.asyncRendering = false
app.usdrt.scene_delegate.geometryStreaming.enabled = false
rtx.hydra.geometrySyncLoads = true
```

**Comment by lilleyse** [comment]
This was autoclosed in https://github.com/CesiumGS/cesium-omniverse/pull/265 (my mistake for writing `fix #255` in that PR summary)

**Comment by lilleyse** [comment]
This should be fixed in the next official Kit release. We'll keep this issue open until then.
